packages/lib-classifier/src/plugins/drawingTools/experimental/components/TranscriptionLine/TranscriptionLine.stories.js

```diff
@@ -7,25 +7,18 @@ import { TranscriptionLineTool } from '@plugins/drawingTools/models/tools'
 import { DrawingToolRoot } from '@plugins/drawingTools/components'
 import TranscriptionLine from './TranscriptionLine'
 
-
 class DrawingStory extends Component {
-  constructor () {
-    super()
-    this.svg = React.createRef()
-  }
-
   render () {
     const { children, mark, tool } = this.props
-    const { svg } = this
+
     return (
-      <svg viewBox='0 0 300 400' height={300} width={400} ref={this.svg}>
+      <svg viewBox='0 0 300 400' height={300} width={400}>
```

Comment:
I think this needs to be wrapped in `SVGContext.Provider`, now that `svg` is passed in context and not as a ref.

Reply:
The mark doesn't use the SVG Context, so is that necessary?

Reply:
Anything that's draggable does use the SVG context.

Reply:
In this case, `DrawingToolRoot` and the drag handles are the context consumers.

Reply:
The story is importing the undecorated `DrawingToolRoot`, so it's not decorated by `draggable` and itself isn't a consumer of the SVG context nor is the mark. I may be missing something though, so pointing to a specific line would be helpful.

Reply:
That's odd. I wonder why it does that? Either way, the mark's drag handles are draggable so watch out for context errors from those.

Reply:
This is actually importing the decorated `DrawingToolRoot` from `drawingTools/components`.

Reply:
#1410 updates `DrawingToolRoot` to export the undecorated component for testing etc.

Reply:
Trying out other marks in #1413, dragging throws `TypeError: svg is not defined` but nothing crashes so I think the default empty context is ok here.